### PR TITLE
Add new extensions to extensions README

### PR DESCRIPTION
**Pull request by nmielnik**
Adding new external extensions into the extensions README, checking with owners to make sure they're ok with this before merging and adding to the landing page.

@cruzanmo any objections with us linking [MediumEditor Phrase](https://github.com/nymag/medium-editor-phrase) in the documentation and on our [Landing Page](https://yabwe.github.io/medium-editor/#extensions)?

@davideanderson any objections with us linking to [MediumEditor Toolbar States](https://github.com/davideanderson/medium-editor-toolbar-states) in the documentation and on our [Landing Page](https://yabwe.github.io/medium-editor/#extensions)?

@dazorni any objections with us linking to [MediumEditor AutoFocus](https://github.com/dazorni/medium-editor-autofocus) in the documentation?

@jawish any objections with us linking to [MediumEditor Thaana Keyboard](https://github.com/jawish/medium-editor-thaanakbd) in the documentation and on our [Landing Page](https://yabwe.github.io/medium-editor/#extensions)?

@epascarello any objections with us linking to [MediumEditor Merge Fields Plugin](https://github.com/epascarello/merge-fields-plugin-for-medium-editor) in the documentation?

@mgilson any objections with us linking to [MediumEditor Google Docs Anchor Preview](https://github.com/patternhq/MediumTools/tree/master/GdocMediumAnchorPreview) in the documentation?

Unless you fellas object, I'm hoping to add this all in in the next day or 2.  Thanks for sharing your extensions!

### CODE REVIEW

**Comment by dazorni** [comment]
Sounds cool 👍

**Comment by mgilson** [comment]
Works for me.

**Comment by cruzanmo** [comment]
+1 Sounds good.

File: src/js/extensions/README.md [hunk 1]
```diff
@@ -125,7 +132,10 @@
   * Supports adding [handsontable](https://handsontable.com/) spreadsheets to MediumEditor.
 * [MediumEditor Lists](https://github.com/mkawczynski07/medium-editor-list)
   * Adds a "Add Paragraph" button which allows for inserting customized paragraphs to MediumEditor
-
+* [MediumEditor Embed Button](https://github.com/orhanveli/medium-editor-embed-button)
+  * oEmbed based embedding button extension to add rich embeds to your document.
+* [MediumEditor Phrase](https://github.com/nymag/medium-editor-phrase)
+  * Phrase button for Medium Editor.
```
Comment (cruzanmo) [inline src/js/extensions/README.md]: medium-editor-phrase was already added to line 129.

Comment (nmielnik) [inline src/js/extensions/README.md]: Oops, thanks for pointing that out 😄

**Pull request merged**